### ISSUE TITLE
Version 0.3.0 breaks the Web build. 0.2.2 works OK

### ISSUE THREAD

**Issue by tomekit** [leanflutter/launch_at_startup#29]
After updating from `0.2.2` to `0.3.0` web build fails with:
```
../../../.pub-cache/hosted/pub.dev/launch_at_startup-0.3.0/lib/src/launch_at_startup.dart:39:34: Error: The method 'isRunningInMsix' isn't defined for the class 'LaunchAtStartup'.
 - 'LaunchAtStartup' is from 'package:launch_at_startup/src/launch_at_startup.dart' ('../../../.pub-cache/hosted/pub.dev/launch_at_startup-0.3.0/lib/src/launch_at_startup.dart').
Try correcting the name to the name of an existing method, or defining a method named 'isRunningInMsix'.
      if (packageName != null && isRunningInMsix(packageName)) {
                                 ^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/launch_at_startup-0.3.0/lib/src/launch_at_startup.dart:40:28: Error: The method 'AppAutoLauncherImplWindowsMsix' isn't defined for the class 'LaunchAtStartup'.
 - 'LaunchAtStartup' is from 'package:launch_at_startup/src/launch_at_startup.dart' ('../../../.pub-cache/hosted/pub.dev/launch_at_startup-0.3.0/lib/src/launch_at_startup.dart').
Try correcting the name to the name of an existing method, or defining a method named 'AppAutoLauncherImplWindowsMsix'.
        _appAutoLauncher = AppAutoLauncherImplWindowsMsix(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```                           

Downgrading library to `0.2.2` solves the problem.

This happens at least on: `Flutter (Channel stable, 3.22.1, on Ubuntu 24.04 LTS 6.8.0-31-generic, locale en_US.UTF-8)`

**Comment by emile-o** [comment]
I think it's because the following was added in 'lib/src/launch_at_startup.dart' for WindowsMsix support

`if (packageName != null && isRunningInMsix(packageName)) {
        _appAutoLauncher = AppAutoLauncherImplWindowsMsix(
          appName: appName,
          appPath: appPath,
          packageName: packageName,
          args: args,
        );
        return;
      }`

and it's importing 'app_auto_launcher_impl_windows_noop.dart' when as web because of this

`import 'package:launch_at_startup/src/app_auto_launcher_impl_windows.dart'
    if (dart.library.html) 'app_auto_launcher_impl_windows_noop.dart';`

so try stubbing the class and function in 'app_auto_launcher_impl_windows_noop.dart' with this

`class AppAutoLauncherImplWindowsMsix extends AppAutoLauncherImplNoop {
  AppAutoLauncherImplWindowsMsix({
    required String appName,
    required String appPath,
    required String packageName,
    List<String> args = const [],
  });
}

bool isRunningInMsix(String packageName) => false;`

**Comment by lijy91** [comment]
Fixed on v0.3.1
